types/ckeditor__ckeditor5-engine/src/view/stylesmap.d.ts

```diff
@@ -494,10 +512,10 @@ export class StylesProcessor {
 }
 
 export interface BoxSides {
```

Comment:
why this specific change, allowing undefined?
https://github.com/ckeditor/ckeditor5/blob/0079571382619493de5a7ec83ffd3a17a0eabaae/packages/ckeditor5-engine/src/view/stylesmap.js#L914-L938

Reply:
Here upstream return all properties set as `undefined`.

https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-engine/src/view/styles/utils.js#L168

Reply:
txh!